classes/task/PKPUsageStatsLoader.php

```diff
@@ -326,10 +322,12 @@ protected function processFile(string $filePath): bool|int
     {
         $loadId = basename($filePath);
         $month = substr($loadId, -12, 6);
-
-        // Check if the log file is an old log file and if the stats for the month already exist
-        if (!$this->isDateValid($loadId) || !$this->isMonthValid($loadId, $month)) {
-            return self::FILE_LOADER_RETURN_TO_STAGING;
+        // if the file is not being reprocessed using the CLI tool
+        if (!in_array($loadId, $this->getOnlyConsiderFiles())) {
```

Comment:
@NateWr, it is actually only this line added to enable the reprocessing of the old log files -- so that the date of the file is not checked if the CLI script is used and thus the _onlyConsiderFiles set.

Reply:
Ok, makes sense! I didn't notice anything else in the code, though I haven't tested it myself. Happy for you to merge it. Or, if you want me to do a more thorough review, leave me some testing steps and I can try it out on Tuesday.

Reply:
For testing you would need that other issue (for conversion of old formats into the new one) that is still in review... I tested it today...
So I will merge... :-)